iosHackthonApp/Modules/Features/MealList/Tests/MealListTests/MealListTests.swift

```diff
@@ -0,0 +1,15 @@
+import XCTest
+@testable import MealList
+
+final class MealListTests: XCTestCase {
```

Comment:
Need to add the tests to the scheme, but after that any new test functions are automatically added to the scheme just like any other test target